src/components/ATATFileUpload.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div v-cloak 
+    @dragenter="onDragEnter" 
+    @drop.prevent="addDropFile" 
+    @dragover.prevent>
+     <v-file-input
+      ref="atatFileUpload"
+      :id = "id + 'FileUpload'"
+      :class="[{'v-text-field--is-hovering' : isHovering},'atat-file-upload']"
+      multiple
+      prepend-icon=""
+      accept="application/pdf,application/vnd.ms-excel"
+      :truncate-length="truncateLength"
+      :clearable="false"
+      @change="fileUploadChanged"
+    >
+      <template v-slot:prepend-inner>
+        <div class="content text-center">
+          <v-icon class="icon-60 mt-10 mb-5">upload_file</v-icon>
+          <h2>Drag and Drop</h2>
+          <p class="mt-1 mb-4 d-flex justify-center">your file here or 
+            <v-btn 
+              class="link-button inline-text" 
+              @mousedown="fileUploadClicked">
+                browse to upload
+              </v-btn>
```

Comment:
pls change from `v-btn` to `a` - unusual hover state. Also add ID for testing
```            
            <a
              role="button"
              id="BrowseToUpload"
              class="_text-link ml-1" 
              @mousedown="fileUploadClicked"
            >
                browse to upload
            </a>
```

Reply:
- [x] Done

Reply:
correct - no `href` if `role="button"`